### PR TITLE
details page design corrections

### PR DESCRIPTION
**Pull request by batsy3**

### CODE REVIEW

File: lib/screens/details_page/details.dart [hunk 1]
```diff
@@ -32,267 +30,315 @@ class Details extends StatefulWidget {
 class _DetailsState extends State<Details> {
   @override
   Widget build(BuildContext context) {
-  String price = widget.price;
+    var additions = ['blue cheese', 'mushroom'];
+    String val = 'blue cheese';
+    String price = widget.price;
```
Comment (bupemulenga13) [inline lib/screens/details_page/details.dart:35]: @batsy3 are we not getting data from the api? Do we need these static values here?

File: lib/screens/details_page/details.dart [hunk 2]
```diff
@@ -32,267 +30,315 @@ class Details extends StatefulWidget {
...
-                                const Text(
-                                  'Description',
-                                  style: TextStyle(
-                                      fontWeight: FontWeight.bold,
-                                      fontSize: 20),
-                                ),
-                              ],
+                          children: [
+                            const Text(
+                              'omg cheese cheese cheese why cheese ahhhhhhhhh cheeseeee echeeeeeeeessseeeeee ',
```
Comment (bupemulenga13) [inline lib/screens/details_page/details.dart:138]: @batsy3 same applies to this, i expect to see variable names representing values coming from the api and not static dummy text.

File: lib/screens/details_page/details.dart [hunk 3]
```diff
@@ -32,267 +30,315 @@ class Details extends StatefulWidget {
...
+                              blurRadius: 3,
+                              spreadRadius: -3,
+                              offset: Offset(0.84, 5))
+                        ],
+                        borderRadius: BorderRadius.all(Radius.circular(40))),
+                    child: const Expanded(
+                      child: ingredient_images(),
+                    ),
+                  ),
+                  // Multipleselect(),
```
Comment (bupemulenga13) [inline lib/screens/details_page/details.dart:199]: @batsy3 kindly remove all unwanted/unused comments.

File: lib/screens/scanning_page/components/https_service.dart [hunk 4]
```diff
@@ -55,7 +49,7 @@ class _HttppState extends State<Httpp> {
   }
 }
 
-const String postUrl = "http://192.168.40.44:8007/api/product";
+const String postUrl = "http://127.0.0.1:8007/api/product";
 Future<Product> _getdata(String? id) async {
   String url = postUrl + '/$id';
```
Comment (bupemulenga13) [inline lib/screens/scanning_page/components/https_service.dart:54]: @batsy3 this is a repeatition of code and could be done in one line for example String url = `http://addresss/api/product/${id}`

File: lib/screens/scanning_page/components/qr_code_scanner.dart [hunk 5]
```diff
@@ -159,4 +156,7 @@ class _QRCodeScannerState extends State<QRCodeScanner> {
     controller?.dispose();
     super.dispose();
   }
+
+  // Widget buildResult() =>
+  //     Text(result != null ? 'Result : ${result!.code},' : 'scan');
```
Comment (bupemulenga13) [inline lib/screens/scanning_page/components/qr_code_scanner.dart:161]: @batsy3 kindly remove unwanted code comments/blocks

**Review by bupemulenga13 — COMMENTED** [review]
@batsy3 left a few comments on some files. Kindly resolve conversations before merge to main

File: lib/screens/details_page/details.dart (hunk 1 above)
Comment (batsy3) [reply lib/screens/details_page/details.dart:35]: im doing these in a seperate pr

File: lib/screens/details_page/details.dart (hunk 2 above)
Comment (batsy3) [reply lib/screens/details_page/details.dart:138]: i need to add some other models in the backend for these things and a few others

**Pull request merged**